static/app/components/feedback/feedbackItem/replaySection.tsx

```diff
@@ -10,27 +10,43 @@ interface Props {
   replayId: string;
 }
 
+const CLIP_OFFSETS = {
+  durationAfterMs: 0,
+  durationBeforeMs: 20_000,
+};
+
 export default function ReplaySection({eventTimestampMs, organization, replayId}: Props) {
+  const hasUserFeedbackReplayClip = organization.features.includes(
```

Comment:
is this a new feature flag??

Reply:
brand new: https://github.com/getsentry/sentry/pull/64920 !